TablePrinter.py

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env python3
+
+# TablePrinter.py - Take list of lists of strings and display
+# it in well-organized table.
+
+
+'''
+For practice, write a program that does the following.
+
+Table Printer
+Write a function named printTable() that takes a list of lists of strings and displays it in a well-organized table with each column right-justified. Assume that all the inner lists will contain the same number of strings.For example, the value could look like this:
```

Comment:
split it into multiple lines, follow the command-line convention i.e. each line should be of 80 characters.

Reply:
Done